firmware/quintuna/VC/src/io/io_lowVoltageBattery.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "hw_gpio.h"
+<<<<<<< HEAD
+#include "cmsis_os2.h"
+
+extern osSemaphoreId_t bat_mtr_sem;
```

Comment:
instead of raw exposing the semaphore, I woudl suggest to expose a function, let's call it something like `io_lowVoltageBattery_completeAlert()`, which releases the semaphore internally. This also prevents jobs.c from containing freertos code (which is undesirable)